### ISSUE TITLE
Using ES6 spread properties in React Components

### ISSUE THREAD

**Issue by Nurymka** [ricardofbarros/linter-js-standard#91]
Using the spread operator when passing propTypes of one component to another causes the linter to give a parsing error. The app still builds, so it should be a bug in the library.

``` javascript
ComponentOne.propTypes = {
  ...ComponentTwo.propTypes
}
```

<img width="457" alt="screen shot 2016-02-16 at 9 52 26 pm" src="https://cloud.githubusercontent.com/assets/7456305/13101149/baa5e500-d4f8-11e5-8b68-ae76d9fc9685.png">

**Comment by ricardofbarros** [comment]
I was unable to reproduce. should be fixed by now.
